### PR TITLE
add CobourgON recommendations

### LINKED ISSUE #6: Create a file for Cobourg, ON Recommendations
# Cobourg, ON

- [ ] Create a branch
- [ ] Add commit
- [ ] Open a Pull Request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge Pull Request

### PR DESCRIPTION
**Pull request by w-harrison**
closes #6

### CODE REVIEW

File: w-harrison-CobourgON.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Cobourg, ON
+
+### Places to eat
+- Oasis
+- Northumberland Heights
```
Comment (jinoac) [inline w-harrison-CobourgON.md]: What type of food is their specialty?

**Comment by jinoac** [comment]
❓

File: w-harrison-CobourgON.md (hunk 1 above)
Comment (w-harrison) [reply w-harrison-CobourgON.md]: The Oasis has an oyster bar

**Review by jinoac — APPROVED** [review]
Excellent!  👍

**Pull request merged**